test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerTest.cs

```diff
@@ -524,7 +538,11 @@ public class ControllerActionInvokerTest
         public async Task InvokeAction_InvokesActionFilter_ShortCircuit()
         {
             // Arrange
-            var result = new EmptyResult();
+            var result = new Mock<IActionResult>(MockBehavior.Strict);
+            result
+                .Setup(r => r.ExecuteResultAsync(It.IsAny<ActionContext>()))
+                .Returns(Task.FromResult(true))
+                .Verifiable();
```

Comment:
This is good. Can you do a pass to make sure that all of the 'short circuit' unit tests verify the IAR is executed once?